src/scripts/webpack/config.js

```diff
@@ -69,16 +72,26 @@ export default function getConfig(args) {
   const localConfigFilePath = path.resolve(configFolderPath, './local.json');
   const localConfig = readOptionalConfigFile(localConfigFilePath);
 
+  const intermediateConfig = {
+    ...referenceConfig,
+    ...localConfig,
+    bundle: {
+      ...referenceConfig.bundle,
+      ...localConfig.bundle
+    }
+  };
+
   // get Config type
   const ConfigType = getConfigType(configFolderPath);
+
+  // build configuration from env variables
```

Comment:
nitpicking: for clarity, I'd change this comment to something like: `// merge environment config values into intermediateConfig (fileConfig)`